frontend/storage/utils/fetchGameData.ts

```diff
@@ -40,25 +41,20 @@ export async function fetchGameBoxData(gameId: number, imageId: number | undefin
 export async function fetchGameAbilitiesListData(gameId: number) {
 	try {
 		const abilityExamples: Array<AbilityExample> = await ApiRequests.get(`ability-examples`, { gameId: gameId });
-		const idsToFetch: Set<number> = new Set();
-		for (const abilityExample of abilityExamples) {
-			idsToFetch.add(abilityExample.abilityId);
-		}
-		const abilities: Array<Ability> = await ApiRequests.get(`abilities/`, { id: Array.from(idsToFetch.values()) });
+		let idsToFetch: Array<number> = unique(abilityExamples.map(x => x.abilityId));
```

Comment:
If you use the more elaborate example I've provided earlier to extract the specific field from models you won't have to use an intermediary `idsToFetch` and could simplify things by directly extracting to `ApiRequests.get`